diff -urN podman-compose-0.1.5-orig/podman_compose.py podman-compose-0.1.5/podman_compose.py
--- podman-compose-0.1.5-orig/podman_compose.py	2019-09-03 17:38:57.000000000 +0200
+++ podman-compose-0.1.5/podman_compose.py	2020-07-03 09:46:58.780991362 +0200
@@ -981,7 +981,7 @@
 
 @cmd_run(podman_compose, 'stop', 'stop specific services')
 def compose_stop(compose, args):
-    transfer_service_status(compose, args, 'start')
+    transfer_service_status(compose, args, 'stop')
 
 @cmd_run(podman_compose, 'restart', 'restart specific services')
 def compose_restart(compose, args):
